pkg/apis/duck/v1alpha1/subscribable_types_conversion.go

```diff
@@ -89,12 +96,7 @@ func (source *SubscribableTypeStatus) ConvertTo(ctx context.Context, obj apis.Co
 			len(source.SubscribableStatus.Subscribers) > 0 {
 			sink.Subscribers = make([]duckv1beta1.SubscriberStatus, len(source.SubscribableStatus.Subscribers))
 			for i, ss := range source.SubscribableStatus.Subscribers {
-				sink.Subscribers[i] = duckv1beta1.SubscriberStatus{
-					UID:                ss.UID,
-					ObservedGeneration: ss.ObservedGeneration,
-					Ready:              ss.Ready,
-					Message:            ss.Message,
-				}
+				sink.Subscribers[i] = *ss.DeepCopy()
```

Comment:
no need to DeepCopy (you can double check if you want: https://github.com/knative/pkg/blob/master/webhook/conversion.go#L61)